bids/reports/utils.py

```diff
@@ -5,6 +5,7 @@
 """
 from __future__ import print_function
 import logging
+import os
```

Comment:
Extraneous import.

```suggestion
import os
```

Reply:
This on is actually for [line 134](https://github.com/bids-standard/pybids/blob/ae1f2681a21cdefa0cdf43a6da6d3dbf0ce98fad/bids/reports/utils.py#L134)

        seqs += ' ({0})'.format(os.path.sep.join(seq_abbrs))

That calls `os` but was not imported. 

Committed in this PR as this crash only appeared after fixing the [.nii .nii.gz] thing.